source/elements/oneTBB/source/algorithms/functions/parallel_sort_func.rst

```diff
@@ -32,11 +32,13 @@ Function template that sorts a sequence.
 Requirements:
 
 * The ``RandomAccessIterator`` type must meet the `Random Access Iterators` requirements from
-  [random.access.iterators]  and `ValueSwappable` requirements from the [swappable.requirements] ISO C++ Standard section.
+  [random.access.iterators] and `ValueSwappable` requirements from the [swappable.requirements] ISO C++ Standard section.
 * The ``Compare`` type must meet the `Compare` type requirements from the [alg.sorting] ISO C++ Standard section.
 * The ``Container`` type must meet the :doc:`ContainerBasedSequence requirements <../../named_requirements/algorithms/container_based_sequence>` 
   which iterators must meet the `Random Access Iterators` requirements from [random.access.iterators]  
   and `Swappable` requirements from the [swappable.requirements] ISO C++ Standard section.
+* The type of dereferenced ``RandomAccessIterator`` or dereferenced ``Container`` iterator must meet the `MoveAssignable`
+  requirements from [moveassignable] and the `MoveConstructible` requirements from [moveconstructible] ISO C++ Standard sections.
```

Comment:
I think `ISO C++ Standard sections` only in the second part of the sentence can confuse - it may be interpreted as [moveconstructible] sections of the standard. I would either add ISO C++ Standard section into the first part of the sentence:

`The type of ... must meet the MoveAssignable requirements from [moveassignable] section of ISO C++ Standard and
 the MoveConstructible requirements from [moveconstructible] section of ISO C++ Standard`.
 
 or try to combine them together:
 
 `The type ... must meet the MoveAssignable and MoveConstructible requirements from [moveassignable] and [moveconstructible] sections of ISO C++ Standard respectively`
 
 `The type ... must meet the MoveAssignable and MoveConstructible requirements from [utility.arg.requirements] section of ISO C++ Standard`
 
 The last variant seems questionable for me because [utility.arg.requirements] contains both [moveassignable] and [moveconstructible] tables and it can confuse. You can choose the variant you like or rephrase it in your manner:)